packages/stencil/src/components/datacom-feature-highlight/datacom-feature-highlight.tsx

```diff
@@ -0,0 +1,70 @@
+import { Component, Host, h, Prop, State, Element, VNode } from '@stencil/core';
+import { getSvg } from '../../common/images/icon-provider';
+
+export type ImageVariant = 'icon' | 'image';
+
+/**
+ * Feature highlight component is used to display content and actions relating to
+ * single subject.
+ */
+@Component({
+  tag: 'datacom-feature-highlight',
+  styleUrl: 'datacom-feature-highlight.css',
+  shadow: true,
+})
+export class DatacomFeatureHighlight {
+  @Element() host: HTMLElement;
+
+  @Prop() variant: ImageVariant = 'icon';
```

Comment:
Rather than having a variant prop, you can detect which to show based on whether your "icon" or "imageUrl" props have been populated. And if both happen to be populated, you can make a decision on which would have priority (maybe image).

Reply:
I developed it before as you suggested. But designers want the variant prop which to show the icon or image.

Reply:
We really shouldn't be modifying our components (like introducing props) just for the sake of storybook. If they're really needing that variant control, you can look to add logic into the story to maybe hide and show a icon or imageUrl input based on that variant control's input. (For an example of conditional controls, have a look at the rating or avatar stories.)

But at the end of the day, I think that variant prop should be removed as it doesn't add any significant value to the component.

Reply:
Thanks for review. I have updated the component and variant renders optionally.

Reply:
Looking at the latest changes, I think you've misunderstood what I was suggesting. I'm asking you to remove the variant prop entirely from the component and have it rely solely on the icon and imageUrl props to render what's needed. (Which is how you implemented it originally as per what you said above.)

Then in storybook, you can leave the storybook variant control, but based on whether it has an "icon" or "image" value, you can conditionally show either the icon or imageUrl storybook controls respectively.

I believe this readonly prop you've introduced is not required and only complicates things further.

Reply:
Thanks for reviewing the component. I have updated the component with the following changes. 
- Removed the variant logic from the component and managed it in the storybook.
- Updated the globe SVG icon.